packages/manager/modules/pci/src/projects/project/storages/databases/add/add.html

```diff
@@ -48,24 +48,38 @@ <h2 id="plan" data-translate="pci_database_add_step2_title"></h2>
                 data-plans="$ctrl.model.engine.selectedVersion.plans"
                 data-selected-plan="$ctrl.model.plan"
                 data-user="$ctrl.user"
-                on-change="$ctrl.onPlanChanged(selectedPlan)"
+                data-on-change="$ctrl.onPlanChanged(selectedPlan)"
+                data-show-monthly-prices="$ctrl.showMonthlyPrices"
             ></database-plans-list>
 
             <!-- Step 3 -->
             <h2 id="region" data-translate="pci_database_add_step3_title"></h2>
             <pci-project-regions-list
                 data-regions="$ctrl.model.plan.regions"
                 data-selected-region="$ctrl.model.region"
-                on-change="$ctrl.onRegionChanged(region)"
+                data-on-change="$ctrl.onRegionChanged(region)"
             ></pci-project-regions-list>
 
             <!-- Step 4 -->
             <h2 id="flavor" data-translate="pci_database_add_step4_title"></h2>
+            <p data-translate="pci_database_add_select_flavor"></p>
+            <database-flavors-list
+                data-flavors="$ctrl.model.region.availableFlavors"
+                data-selected-flavor="$ctrl.model.flavor"
+                data-user="$ctrl.user"
+                data-on-change="$ctrl.onFlavorChanged(selectedFlavor)"
+                data-show-monthly-prices="$ctrl.showMonthlyPrices"
+                data-nodes="$ctrl.model.plan.nodesCount"
+            ></database-flavors-list>
```

Comment:
**nitpick:** Could you please add prefix to the component name. the current name is generic.
`pci-projects-project-database-flavors-list`

Reply:
Sure, done here: https://github.com/ovh/manager/pull/8392/commits/da083feb91a6c6c14b561807f0b1b1918acba75e